daniel-robert/server.js

```diff
@@ -134,7 +136,7 @@ app.delete('/articles', function(request, response) {
 });
 
 // COMMENT: What is this function invocation doing?
-// Put your response here...
+// Put your response here... loadDB() is going to create the database and input all the data that you put into it.
```

Comment:
Won't create the database, you already created the database (kilovolt or blog_DB) and directed your app to use it by specifying it in the con-string, think about what does this creates instead?

Reply:
this method specifically creates the table within the DB and then calls the function to load the articles into the table.